pym/portage/util/writeable_check.py

```diff
@@ -58,7 +58,11 @@ def linux_ro_checker(dir_list):
 				# to the left of the ' - ', after the attr's, so split it there
 				mount = line.split(' - ', 1)
 				_dir, attr1 = mount[0].split()[4:6]
-				attr2 = mount[1].split()[2]
+				# check for situation with invalid entries for /home and /root in /proc/self/mountinfo
+				try:
+					attr2 = mount[1].split()[2]
+				except IndexError:
+					attr2 = mount[1].split()[1]
```

Comment:
The mount array might not contain 2 elements, depending on the input, so mount[1] is still unsafe.